src/Entity/CitizenProject.php

```diff
@@ -319,6 +320,12 @@ public function approved(string $timestamp = 'now'): void
     public function update(string $name, NullablePostAddress $address): void
     {
         $this->setName($name);
+        $this->setSubtitle($subtitle);
```

Comment:
lots of variables are undefined here

Reply:
Thx, it's a mistake, I updated this method in another PR. :+1: